profiles/govcms_saas.yml

```diff
@@ -88,3 +88,23 @@ checks :
     max_size : 5
     warning_size : 2
   \SiteAudit\Check\Phantomas\InPage404s : {}
+
+  # Field specific checks.
+  \SiteAudit\Check\D7\EntityReferenceAutocomplete:
+    threshold: 100
+
+  # Module configuration checks.
+  \SiteAudit\Check\D7\ModuleSettings:
+    label: 'Login Security'
+    machine_name: 'login_security'
+    enabled: true
+    settings:
+      \SiteAudit\Settings\VariableIterator:
```

Comment:
Plugable iterators allow us to define a different way to check settings if we need to. Say; a module maintains a configuration table separately- we could define another iterator to query DBs as opposed to variable checks.